lib/vector-core/src/transform/config.rs

```diff
@@ -9,10 +9,10 @@ pub enum DataType {
     Metric,
 }
 
-#[derive(serde::Deserialize, serde::Serialize, Debug)]
+#[derive(serde::Deserialize, serde::Serialize, Clone, Debug)]
 pub enum ExpandType {
-    Parallel,
-    Serial,
+    Parallel { aggregates: bool },
+    Serial { alias: bool },
```

Comment:
Could leave get a small comment about what they do?  Specifically the fields.  As an outsider, from, reading the pipelines RFC, I assume `Parallel` and `Serial` are related to the route and linear modes for pipelines, but it's hard to tell without any comments.